go/genesis/genesis_test.go

```diff
@@ -178,13 +170,13 @@ func TestGenesisSanityCheck(t *testing.T) {
 		},
 		AdmissionPolicy: registry.RuntimeAdmissionPolicy{
 			EntityWhitelist: &registry.EntityWhitelistRuntimeAdmissionPolicy{
-				Entities: map[signature.PublicKey]bool{
-					validPK: true,
+				Entities: map[signature.PublicKey]registry.EntityWhitelistConfig{
+					validPK: {MaxNodes: make(map[node.RolesMask]uint16)},
```

Comment:
I think a nil map should behave exactly the same so you could do:
```suggestion
					validPK: {},
```